src/browser/components/molecules/Task/index.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react'
+import Layout from '../../molecules/Layout/index'
+import styles from './index.css'
+const moment = require('moment')
+
+
+
+export default class Task extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      complete: false
+    }
+    this.isCompleteHandler = this.isCompleteHandler.bind(this)
+  }
+
+  isCompleteHandler( event ){
+    const completeTaskDetails = {
+      method: 'POST',
+      mode: 'cors',
+      headers: new Headers({
+        'Accept': 'application/json, application/xml, text/plain, text/html, *.*',
+        'Content-Type': 'application/json',
+         }),
+      credentials: 'same-origin',
+      body: JSON.stringify({is_complete: true})
+    }
+    fetch( `http://noob.learnersguild.dev/api/task/${this.props.id}`, completeTaskDetails )
+    .then( resultPromise => {
```

Comment:
can we rename this to `tasksPromise`? the variable name should indicate what it points to